packages/provider/src/tests/runProviderTest.ts

```diff
@@ -7,1803 +7,1805 @@ export function runProviderTest<
   Options extends JoshProvider.Options = JoshProvider.Options,
   CleanupProvider extends JoshProvider = JoshProvider
 >(options: ProviderTestOptions<Provider, Options, CleanupProvider>): void {
-  const { providerConstructor: Provider, providerOptions = {}, cleanup } = options;
+  const { providerConstructor: Provider, providerOptions = {}, cleanup, serialization } = options;
 
-  describe(Provider.prototype.constructor.name, () => {
-    describe('is a class', () => {
-      test(`GIVEN typeof ${Provider.prototype.constructor.name} THEN returns function`, () => {
-        expect(typeof Provider).toBe('function');
-      });
+  for (const serialize of serialization === false ? [true] : [true, false]) {
```

Comment:
```suggestion
  for (const serialize of serialization ? [true, false] : [true]) {
```

Reply:
Uh, well serialization being set to false would only want 1 test, otherwise it would test serialization, hence why the naming of the original made it easier to understand

Reply:
All I did here was invert the tenary statement

Reply:
Right but serialization is default undefined

Reply:
But don't we want it to default check the serialization?

Reply:
Not really tbh